products/firefox_android/testing.md

```diff
@@ -25,9 +25,9 @@ It is important that the people testing the localization be not only the one(s)
 
 Once the Fennec Project Manager (currently **delphine** at **mozilla** dot **com**) sees that you’ve done at least one cycle on a single-locale build, that nothing breaks during sign-offs, that you’ve continued keeping your work up to date, and that you have been TESTING it - l10n-drivers will consider adding your language to the multi-locale builds. This means that - congratulations! - your locale will now officially be shipping on Firefox for Android.
 
-Once your locale is added to the multi-locale builds, it will "ride the trains" like for Firefox Desktop. It will first ship on Aurora in the Play Store, then 6 weeks later it will ship on Beta, and 6 weeks after that, it will arrive on Release.
+Once your locale is added to the multi-locale builds, it will "ride the trains" like for Firefox Desktop. It will first start shipping on Nightly on the Play Store, then once next merge day arrives it will ship on Beta, and again the following merge day, it will arrive on Release (cycles are usually 6 or 8 weeks). You can check all upcoming merge days on the [Rapid Release Calendar](https://wiki.mozilla.org/RapidRelease/Calendar).
 
-Once you start shipping in the Play Store, it makes sense to stop testing on single-locale builds. It’s strongly suggested you now start testing and using the Aurora Play Store version, since this is the branch that you localize and work on. It’s good to have other members in the community use the beta and release version too though, in case anything breaks with updates.
+Once you start shipping in the Play Store, it makes sense to stop testing on single-locale builds. It’s strongly suggested you now start testing and using the [Nightly Play Store version](https://play.google.com/store/apps/details?id=org.mozilla.fennec_aurora), since this is the branch that you localize and work on. Don’t be fooled by the "aurora" in that URL - it’s still redirecting to our Fennec Nightly builds. Note that it’s good to have other members in the community use the beta and release version too, in case anything breaks with updates.
```

Comment:
Nit: “aurora”

"it’s still redirecting to our Fennec Nightly builds" is not correct, more something like: 

```
Don’t be fooled by the “aurora” in that URL: the existing package ID for Aurora was re-used for Fennec Nightly builds shipping on Play Store.
```